pkgs/os-specific/darwin/apple-source-releases/Libsystem/default.nix

```diff
@@ -3,10 +3,13 @@
   removefile, libresolv, Libnotify, libplatform, libpthread, mDNSResponder, launchd, libutil, version }:
 
 appleDerivation rec {
-  phases = [ "unpackPhase" "installPhase" ];
```

Comment:
Maybe we should get rid of these in the other apple stuff, it can be pretty confusing when stuff like multiple outputs don't work if you don't think of this immediately.

Reply:
Yes! I think they can all be either removed or replaced with ```dontBuild = true``` & ```dontConfigure = true```. I'm never sure whether they're actually needed though.

Reply:
Yeah, this was all clueless @copumpkin right at the beginning of his Nix career not knowing what he's doing. And yes, he occasionally speaks of himself in the third person.